README.md

```diff
@@ -1,339 +1,103 @@
-  MAkGA1UEBhMCREUxCzAJBgNVBAgMAkJZMRIwEAYDVQQHDAlXdWVyemJ1cmcxFDAS
-  BgNVBAoMC0V4YW1wbGUgb3JnMSIwIAYDVQQDDBlzc28ubG9jYWwuZGV2LmJpdGdy
-  aWQubmV0MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQDHPZwU+dAc76yB6bOq
-  0AkP1y9g7aAi1vRtJ9GD4AEAsA3zjW1P60BYs92mvZwNWK6NxlJYw51xPak9QMk5
-  qRHaTdBkmq0a2mWYqh1AZNNgCII6/VnLcbEIgyoXB0CCfY+2vaavAmFsRwOMdeR9
-  HmtQQPlbTA4m5Y8jWGVs1qPtDQIDAQABo1MwUTAdBgNVHQ4EFgQUeZSoGKeN5uu5
-  K+n98o3wcitFYJ0wHwYDVR0jBBgwFoAUeZSoGKeN5uu5K+n98o3wcitFYJ0wDwYD
-  VR0TAQH/BAUwAwEB/zANBgkqhkiG9w0BAQsFAAOBgQA25X/Ke+5dw7up8gcF2BNQ
-  ggBcJs+SVKBmPwRcPQ8plgX4D/K8JJNT13HNlxTGDmb9elXEkzSjdJ+6Oa8n3IMe
-  vUUejXDXUBvlmmm+ImJVwwCn27cSfIYb/RoZPeKtned4SCzpbEO9H/75z3XSqAZS
-  Z1tiHzYOVtEs4UNGOtz1Jg==
-  -----END CERTIFICATE-----
-  ```
-- cn `urn:oid:2.5.4.3`
-- email `urn:oid:0.9.2342.19200300.100.1.3`
-
-### Environment based SSO
-
-A simple approach to test environment based SSO with the user_saml app is to use apache basic auth with the following configuration:
-
-```
-
-<Location /login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /index.php/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /index.php/apps/user_saml/saml/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-<Location /apps/user_saml/saml/login>
-	AuthType Basic
-	AuthName "SAML"
-	AuthUserFile /var/www/html/.htpasswd
-	Require valid-user
-</Location>
-```
-
-## Fulltextsearch
-
-```
-docker-compose up -d elasticsearch elasticsearch-ui
-```
-
-- Address for configuring in Nextcloud: `http://elastic:elastic@elasticsearch:9200`
-- Adress to access elastic search from outside: `http://elastic:elastic@elasticsearch.local`
-- Address for accessing the ui: http://elasticsearch-ui.local/
-
-`sudo sysctl -w vm.max_map_count=262144`
-
-
-
-## Object storage
-
-Primary object storage can be enabled by setting the `PRIMARY=minio` environment variable either in your .env file or in docker-compose.yml for individual containers.
-
-```bash
-docker-composer up proxy nextcloud minio
-```
-
-## Development
-
-### OCC
-
-Run inside of the Nextcloud container:
-```
-set XDEBUG_CONFIG=idekey=PHPSTORM
-sudo -E -u www-data php -dxdebug.remote_host=192.168.21.1 occ
-```
-
-### Useful commands
-
-- Restart apache to reload php configuration without a full container restart: `docker-compose kill -s USR1 nextcloud`
-- Access to mysql console: `mysql -h $(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_database-mysql_1) -P 3306 -u nextcloud -pnextcloud`
-- Run an LDAP search: `ldapsearch -x -H ldap://$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_ldap_1) -D "cn=admin,dc=planetexpress,dc=com" -w admin -b "dc=planetexpress,dc=com" -s subtree <filter> <attrs>`
```

Comment:
Where are these gone?

Reply:
I moved some of this documentation to the `setup-xdebug.md` file.

I added a link in the README.md file : https://github.com/arawa/nextcloud-docker-dev/blob/13a001965c2e7472474295aa361fdd26baa0a204/README.md?plain=1#L78

Reply:
do you feel that something needs to be changed ? Don't hesitate to tell me :)